### DISCUSSION TITLE
Image upload in github issue fails due to CORS error

### DISCUSSION THREAD

**Discussion by FieteO** [community/community#36380] [Projects and Issues]
I am not sure if this is the right place to address this, but here is my problem:
Upon trying to upload an image in a github issue, an `OPTIONS` request is made to https://github-production-user-asset-6210df.s3.amazonaws.com/ which fails since the response does not have an `access-control-allow-origin` header.

As far as I can tell, this prevents the image from being uploaded.

**Comment by ossanna16** [comment]
Hey there @FieteO and thank you for asking a good question! I'm glad you found our community 🙂

**Reply by damzog** [reply]
Any findings on this matter? I see a lot of similar questions around. Seems I have the same problem and unfortunately I can't attach a screenshot!
```
36380:1 Access to XMLHttpRequest at 'https://github-production-user-asset-6210df.s3.amazonaws.com/' from origin 'https://github.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
attachment-upload.ts:114 
        
        
POST https://github-production-user-asset-6210df.s3.amazonaws.com/ net::ERR_FAILED 403 (Forbidden)
(anonymous) @ attachment-upload.ts:114
process @ attachment-upload.ts:111
tv @ batch-upload.ts:16
await in tv (async)
(anonymous) @ file-attachment.ts:37
S @ index.js:96
attach @ index.js:164
u @ index.js:246
Show 3 more frames
Show less
```

**Reply by humaizfs** [reply]
@damzog I am also running into this problem. Were you able to find a solution?

**Reply by damzog** [reply]
@humaizfs No. As a workaround I created  a separate gist and added the link only.

**Comment by alleksei37** [comment]
I ran into the same issue since yesterday. We can't attach images to our pull requests due to the CORS error with s3.amazonaws.com
